src/phc2sys.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Synchronize system clock or PTP hardware clock (PHC)
+Documentation=man:phc2sys
+After=ptp4l@%i.service
```

Comment:
You can not use the %i parameter with a regular service.
Create a template service: phc2sys@.service
```
[Unit]
Description=Synchronize system clock or PTP hardware clock (PHC) with %i
Documentation=man:phc2sys
Requires=ptp4l@%i.service
After=ptp4l@%i.service
Before=time-sync.target
```

Reply:
I forget to remove the After, we do not use ptp4l because we use ptp_kvm inside the vm